index.html

```diff
@@ -301,6 +301,72 @@ <h3>
           </table>
         </section>
 
+        <section id="normative">
+          <h3>
+            Registries
+          </h3>
+          <p>
+The Working Group will deliver a set of registries on the [W3C Registry
+Track](https://www.w3.org/2021/Process-20211102/#registries) to
+support extension points in the above normative deliverables. The
+Working Group expects to produce the following registries, but it will
+adjust this list as needed to support the normative deliverables:
+          </p>
+          <table>
+            <tr>
+              <th>Registry</th>
+              <th>Description</th>
+              <th>Based on</th>
+            </tr>
+            <tr>
+              <td>
+Verifiable Credential Properties Registry
+              </td>
+              <td>
+A registry of extensions to properties used in the Verifiable Credentials Data
+Model specification.
+              </td>
+              <td>
+<a href="https://w3c-ccg.github.io/vc-extension-registry/">Verifiable Credential Extensions Registry</a>
+              </td>
+            </tr>
+            <tr>
+              <td>
+Cryptographic Container and Suite Registry
+              </td>
+              <td>
+A registry of cryptographic container formats and cryptographic suites that
+are useful in conjunction with specifications managed by this Working Group.
+              </td>
+              <td>
+<a href="https://w3c-ccg.github.io/security-vocab/#classes">Classes in CCG Security Vocabulary</a>
```

Comment:
Again, please remove all proposed registries that cite CCG documents.  We should control our own destiny and only create registries for things that this working group creates.  Our outputs should have no dependencies on CCG documents.

Reply:
The CCG documents are described here as inputs, rather than outputs, so I think that matches what you've been asking for. It's plausible to leave out the list of likely registries entirely (although I think it's useful), but mentioning CG documents matches the precedent in the WebApps charter.